### DISCUSSION TITLE
Markup what has been reviewed in Splink comparison viewer

### DISCUSSION THREAD

**Discussion by ianiredan** [moj-analytical-services/splink#1346] [Q&A] [answered]
Hello, 
I have used the Splink tool in a few projects comparing datasets containing a few million records, and the tool has been very intuitive and easy to understand the matching model through the use of the interactive outputs, so my thanks to the developers.

My question is in regards to the Splink comparison viewer. I have found this tool very useful for reviewing the different matching combinations, however it can be easy to lose track of what has already been reviewed. 

Is it possible to apply some sort of markup on the comparison vectors bar chart within the comparison viewer .html file to denote which vectors have and have not been reviewed?

I have managed to apply some filtering by regenerating the results from the comparison viewer using information from the discussion under https://github.com/moj-analytical-services/splink/discussions/1089, however I would like to provide the comparison viewer chart to other team members to assist in clerical review of the matching results, and they would not have the technical knowledge to perform this task,

**Comment by RossKen** [comment, marked as answer]
Hi @ianiredan, thanks for your feedback! 

I don't know if we would want to implement this review functionality within the comparison viewer html file, but it is an interesting point so I will go away and think about it.

For model QA more generally, we will be releasing a beta version of a [clerical labelling tool](https://github.com/moj-analytical-services/splink/pull/1208) very soon that will allow users to manually label a sample of record pairs to help generate [performance metrics](https://moj-analytical-services.github.io/splink/demos/07_Quality_assurance.html) which may be more appropriate for less technical team members. Then, you can generate the waterfall charts of any false positives/false negatives to have a closer look at where your model is going wrong as shown [at the bottom of this notebook.](http://127.0.0.1:8000/demos/example_accuracy_analysis_from_labels_column.html)

**Reply by ianiredan** [reply]
Hi @RossKen, thank you for your answer! 

To provide a bit more context for my suggestion, I have found that as I am reviewing the comparison viewer html file, I may come across combinations of some comparison levels could result in a higher likelihood of false positives, so I would want to flag these sets for further clerical review, even if they have a high matching score. To track these, I have maintained a separate spreadsheet of the gamma values within the comparison vectors, along with a note. 

Thank you for providing information on the clerical labelling tool. That looks like it will be useful; I look forward to trying that out.
